wasm/README.md

```diff
@@ -46,13 +39,43 @@ WebSocket implementation. Two of such modules are [WebSocket](https://www.npmjs.
 (provides a custom implementation) and [isomorphic-ws](https://www.npmjs.com/package/isomorphic-ws) 
 (built on top of the ws WebSocket module).
 
-You can use the following shims:
 
-```js
-// `websocket` module
+## Loading in a Web App
+
+```html
+<html>
+    <head>
+        <script type="module">
+            import * as kaspa_wasm from './kaspa/kaspa-wasm.js';
+            (async () => {
+                const kaspa = await kaspa_wasm.default('./kaspa/kaspa-wasm_bg.wasm');
+            })();
+        </script>
+    </head>
+    <body></body>
+</html>
+```
+
+## Loading in a Node.js App
+
+```javascript
+// W3C WebSocket module shim
 globalThis.WebSocket = require('websocket').w3cwebsocket;
-// `isomorphic-ws` module
-globalThis.WebSocket = require('isomorphic-ws');
+
+let {RpcClient,Encoding,init_console_panic_hook,defer} = require('./kaspa-rpc');
+// init_console_panic_hook();
```

Comment:
Is this intended to be commented out? If it is, you should say when it should be uncommented.

Reply:
Thanks for noting this. This was written months ago and I completely overlooked that functions have been renamed to `initConsolePanicHook()` to have camel case.  Updated, added comments and added `initBrowserPanicHook()` into the mix.